src/shared/src/components/Select.tsx

```diff
@@ -0,0 +1,71 @@
+import { makeStyles } from '@material-ui/core/styles';
+import cn from 'classnames';
+import * as React from 'react';
+import { AltinnAppTheme } from '../theme';
+import '../../../altinn-app-frontend/src/styles/shared.css';
+
+const useStyles = makeStyles({
+  select: {
+    fontSize: '1.6rem',
+    '&:focus': {
+      outline: `2px solid ${AltinnAppTheme.altinnPalette.primary.blueDark}`,
+    },
+  },
+});
+
+const optionStyle = {
+  display: 'none',
+};
+interface ISelectProps {
+  id: string;
+  onChange: (event: React.ChangeEvent<HTMLSelectElement>) => void;
+  onBlur?: (event: React.FocusEvent<HTMLSelectElement>) => void;
+  options: { value: string; label: string }[];
+  value: string;
+  disabled?: boolean;
+  error?: boolean;
+  fullWidth?: boolean;
+  className?: string;
+}
+
+const Select = ({
+  id,
+  onChange,
+  onBlur,
+  options,
+  value,
+  disabled = false,
+  error = false,
+  fullWidth = false,
+  className,
+}: ISelectProps) => {
+  const classes = useStyles();
+  return (
+      <select
+        id={id}
+        value={value}
+        disabled={disabled}
+        className={cn(
+          classes.select,
+          'custom-select a-custom-select',
+          {
+            'validation-error': error,
+            'disabled !important': disabled,
+          },
+          className,
+        )}
+        style={{ width: fullWidth ? '100%' : 'unset' }}
+        onChange={onChange}
+        onBlur={onBlur}
+      >
+        <option style={optionStyle} />
```

Comment:
why is there a hidden first option here?

Reply:
I just copied this implementation from the `DropDownComponent.tsx`. If the value for the dropdown is empty this hidden option will be displayed. If the hidden option is removed the select will display the first option in the list. 

If we make the option visible, we are able to select an empty value after we have selected something.

This is an issue btw: https://github.com/Altinn/app-frontend-react/issues/17